README.rst

```diff
@@ -111,10 +111,25 @@ async/await fixtures
 pytest fixture semantics of setup, value, and teardown.  At present only
 function scope is supported.
 
+Note: in pytest-twisted version 0.12, you must *call*
```

Comment:
```suggestion
Note: You must *call*
```
I feel like a version here is just extra work to maintain.  Adding the version if/when it changes to point out the transition would be good.

Reply:
Agreed.